From dad230792b046c711f4e491cfdbabda58862ee78 Mon Sep 17 00:00:00 2001
From: Martin Kutlak <mkutlak@redhat.com>
Date: Wed, 12 Dec 2018 16:12:23 +0100
Subject: [PATCH] shellcheck: Suppress shellcheck warning SC1090

ShellCheck is not able to include sourced files from paths that are determined at runtime.
The file will not be read, potentially resulting in warnings about unassigned variables and similar.

If you don't care that ShellCheck is unable to account for the file, specify
"# shellcheck source=/dev/null".

Signed-off-by: Martin Kutlak <mkutlak@redhat.com>
---
 src/plugins/abrt-action-analyze-ccpp-local.in | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/plugins/abrt-action-analyze-ccpp-local.in b/src/plugins/abrt-action-analyze-ccpp-local.in
index 92593437..9144c0e6 100644
--- a/src/plugins/abrt-action-analyze-ccpp-local.in
+++ b/src/plugins/abrt-action-analyze-ccpp-local.in
@@ -26,8 +26,9 @@ if $INSTALL_DI; then
     EXTRA_ARGS=
     for osrel in "${DUMP_DIR:-.}/os_info_in_rootdir" "${DUMP_DIR:-.}/os_info"
     do
-        if [ -e $osrel ]; then
-            . $osrel
+        if [ -e "$osrel" ]; then
+            # shellcheck source=/dev/null
+            . "$osrel"
             if [ -n "$VERSION_ID" ]; then
                 EXTRA_ARGS="--releasever=$VERSION_ID"
                 break
-- 
2.26.2

